crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -147,14 +147,18 @@ def services
         end
 
         # Initiate the services shutdown by calling scripts on all nodes.
-        # For each cluster, it is enough to initiate the shutdown from one node (e.g. founder)
-        NodeObject.find("state:crowbar_upgrade AND pacemaker_founder:true").each do |node|
-          node.ssh_cmd("/usr/sbin/crowbar-shutdown-services-before-upgrade.sh")
-        end
-        # Shutdown the services for non clustered nodes
-        NodeObject.find("state:crowbar_upgrade AND NOT run_list_map:pacemaker-cluster-member").
-          each do |node|
-          node.ssh_cmd("/usr/sbin/crowbar-shutdown-services-before-upgrade.sh")
+        NodeObject.find("state:crowbar_upgrade").each do |node|
```

Comment:
So I placed it to the `node_object`, which I hope is good for now, but should be cleaned up once we have model for nodes that are being upgraded ...